internal/attestation/attestation_test.go

```diff
@@ -125,3 +127,252 @@ func TestProvenanceFromSignature(t *testing.T) {
 		})
 	}
 }
+
+func TestProvenance_Type(t *testing.T) {
+	tests := []struct {
+		name     string
+		expected string
+	}{
+		{
+			name:     "returns correct in-toto statement type",
+			expected: "https://in-toto.io/Statement/v0.1",
+		},
+		{
+			name:     "returns consistent type value",
+			expected: "https://in-toto.io/Statement/v0.1",
+		},
```

Comment:
I might be missing something, but it seems identical to the other one, so I don't think it's useful.

```suggestion
```